crates/bevy_asset/Cargo.toml

```diff
@@ -41,6 +41,7 @@ ron = "0.8"
 serde = { version = "1", features = ["derive"] }
 thiserror = "1.0"
 uuid = { version = "1.0", features = ["v4"] }
+tempfile = "3.10.1"
```

Comment:
This dependency is already part of the graph transitively via `accesskit_winit`, so its explicit inclusion shouldn't be a controversial choice.